etc/mail-annoucement.txt

```diff
@@ -0,0 +1,27 @@
+Hello HTTP working group,
+
+in the past months we have been working on a draft for resumable file uploads using HTTP and want to present our work to you. To briefly recap, resumable uploads allow resuming an upload where it has been stopped. For example, after the connection has been interrupted. A 100 MB upload, which was interrupted after transmitting 70 MB, only needs to transmit the remaining 30 MB.
+
+There is support in HTTP for resumable downloads using the Range header, but no support for resumable uploads so far. There have been attempts to standardize resumable uploads, such as [1], but none completed this journey yet to our knowledge.
+
+We wanted to address this shortage and came up with a draft [2] (the GitHub repository is at [3]). We are interested any feedback on this and see if it is possible to get resumable uploads adopted.
```

Comment:
I think it's missing an "in": "We are interested IN any feedback on this..."

Reply:
Following on from my above suggestion, I would tweak this para to say

"We believe resumable uploads are a common problem and that there is value in a standard resumable upload approach. We've been working on a document [2] [3] that uses HTTP to solve what we believe to be the core problem set, while also allowing for extended use cases. We are bringing this to the list to understand if there is interest in the working group to solve the problem, and whether our document is a good basis for a solution.